### PR TITLE
Bump manpage versions for 1.1.0

### PR DESCRIPTION
**Pull request by rillian**
Quick fix of issues I noticed in #25689.

### CODE REVIEW

File: man/rustc.1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.TH RUSTC "1" "March 2014" "rustc 0.13.0" "User Commands"
+.TH RUSTC "1" "May 2015" "rustc 1.1.0" "User Commands"
```
Comment (steveklabnik) [inline man/rustc.1]: 1.1 will actually be released in June, no?

File: man/rustdoc.1 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.TH RUSTDOC "1" "March 2014" "rustdoc 0.13.0" "User Commands"
+.TH RUSTDOC "1" "May 2015" "rustdoc 1.1.0" "User Commands"
```
Comment (steveklabnik) [inline man/rustdoc.1]: same here

**Comment by rillian** [comment]
Yes. I can match the date to the release if you'd rather. I just used the current date since that's the most recent modification.

**Comment by alexcrichton** [comment]
This PR is targeting the beta branch directly so I'm going to do a direct merge, thanks @rillian!

**Pull request merged**